webpack/assets/javascripts/react_app/components/toast/toastListitem/index.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import Icon from '../../common/Icon';
+import helpers from '../../../common/helpers';
+import CloseButton from '../../common/CloseButton';
+import Timer from '../../../common/Timer';
+import { defaultTimerDelay } from './Toast.consts';
+
+class Toast extends React.Component {
+  constructor(props) {
+    super(props);
+    helpers.bindMethods(this, [
+      'onMouseEnter',
+      'onMouseLeave'
+    ]);
+  }
+
+  componentDidMount() {
+    const { sticky, dismiss, timerDelay } = this.props;
+
+    if (!sticky) {
```

Comment:
q: i assume this is implemented this way before redux implementation, but just wondering, would it be better to have the timer as an action instead?